web-ui/src/main/resources/catalog/locales/en-search.json

```diff
@@ -407,5 +407,7 @@
   "processesTool": "Use a WPS process",
   "wpsGetCapError": "Error while parsing WPS capabilities: ",
   "wpsByUrl": "Search processes",
-  "wpsRecent": "Recent Processes"
+  "wpsRecent": "Recent Processes",
+  "selection.indexing.count": "{{count}} record(s) have been indexed.",
```

Comment:
Indexing is done in new threads, so message should be something like "Indexing started for the 23 records selected." ?